README.rst

```diff
@@ -2,11 +2,141 @@
 community_mailbot
 =================
 
-The ``community_mailbot`` is a friendly bot that tells subscriber's to LSST DM's legacy Mailman email lists about things happening on `community.lsst.org <http://community.lsst.org>`_.
+The ``community_mailbot`` is a friendly bot that tells subscribers to LSST DM's legacy Mailman email lists about things happening on `community.lsst.org <http://community.lsst.org>`_.
+
+The bot likes to be awoken regularly by ``cron``; each time it will ask the Discourse server about latest messages in different categories that the bot tracks (you can set this with a simple ``config.json`` file).
+If there is are new topics, ``community_mailbot`` will send an email the appropriate email lists using its buddy, `Mandrill <http://mandrillapp.com>`_.
+Before the bot goes back to sleep, it keeps note of all the topics it's emailed already in a simple JSON cache file.
+
+Installation
+------------
+
+Create a virtual environment running Python 3.5, then:
+
+.. bash::
+
+   pip install git+https://github.com/lsst-sqre/community_mailbot.git@tickets/DM-3690
+
+Running the community mailbot
+-----------------------------
+
+Keys to the Kingdom
+~~~~~~~~~~~~~~~~~~~
+
+You'll need to get API keys from community.lsst.org and Mandrill.
+Set them to the following environment variables:
+
+* ``$MANDRILL_KEY`` (note, use the API key for the ``community_mailbot`` subaccount)
+* ``$DISCOURSE_KEY`` (the Discourse key should correspond to a user with sufficient permissions)
+* ``$DISCOURSE_USER``
+
+Optionally set ``$COMMUNITY_MAILBOT_CACHE`` to the location where you want the Mailbot to keep track of its topics it has forwarded.
+
+Setup Topic → Email Mapping
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To setup how Discourse categories are mapped to recipient email addresses, you need to know the integer IDs for each relevant category.
+To get these, run the included script:
+
+.. code:: bash
+
+   discourse_categories
+
+If you set the aforementioned environment variables, you won't need to provide any arguments to the script.
+
+Next, create a ``config.json`` file.
+It's a simple hash structure; each key-value pair is the Discourse category ID and a list of email recipients for that category.
+The general format is:
+
+.. code:: text
+
+   {
+       "<id>": [{"email": "<email address>",
+                 "name": "<recipient name>",
+                 "type": "to"}],
```

Comment:
Clarification about the "type" (which is a Mandrill thing, I guess) would be good -- always say "to"? are there other possible values?